tests/test_wagon.py

```diff
@@ -772,13 +695,19 @@ def teardown_method(self, test_method):
         if os.path.isdir('test_env'):
             shutil.rmtree('test_env', ignore_errors=True)
 
-    # TODO: Important before releasing 0.6.0!
-    @pytest.mark.skipif(not os.environ.get('CI'),
```

Comment:
we are now making a (temporary) virtualenv inside this test, so it can run on non-CI as well, without leaving trash around after a test run